readme.md

```diff
@@ -17,8 +17,13 @@ var splitLines = require('split-lines');
 
 splitLines('foo\r\nbar\r\nbaz\nrainbow');
 //=> ['foo', 'bar', 'baz', 'rainbow']
+
+splitLines('foo\r\nbar\r\nbaz\nrainbow', true);
```

Comment:
Can you make it an options object? I don't like boolean arguments.

```js
splitLines('foo\r\nbar\r\nbaz\nrainbow', {preserveNewlines: true});
```

-

I also think `preserveNewlines` would be a better name.